code/src/components/Question3/Question3.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+
+import './Question3.css';
+
+const answers = ['today', 'yesterday', 'last week', "I don't know"];
```

Comment:
Single or double quotation marks, we need to choose :) I just read today that it way more common in React to use single quotation marks, but in the jsx part (the HTML inside of these JS files, for exemple when you add the className attribute, or other attributes), it should be double quotation marks. You can set this so it's done automatically in VS code settings with Prettier. And then you just use the shortcut and your file has automatically the right ones (along with all the good formatting).